backbone/src/main/res/layout/rsb_integer_scale_step_layout.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:padding="8dp">
+
+    <TextView
+        android:id="@+id/integerScaleStepQuestionTitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="@id/reviewStepEditButtonBarrier"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Integer Scale Step Title" />
+
+    <TextView
+        android:id="@+id/integerScaleStepQuestion"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:textColor="#757575"
+        android:textStyle="italic"
+        app:layout_constraintEnd_toEndOf="@id/reviewStepEditButtonBarrier"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/integerScaleStepQuestionTitle"
+        tools:text="On a scale of 1-10 how thirsty are you?" />
+
+    <TextView
+        android:id="@+id/integerScaleStepQuestionAnswer"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:background="#f7f8fa"
+        android:padding="5dp"
+        android:textStyle="bold"
+        android:inputType="number"
```

Comment:
Does this do anything on a TextView? I thought it was an attribute of `Editable` (like an `EditText`).

Reply:
This can be changed to Editable, and that was like the design
https://app.zeplin.io/project/5d8e16c2db3c0d61aac43a5f/screen/5da0bfe5e3c51c2f6ccf234d